steps/azure_container_registry_purge.yml

```diff
@@ -0,0 +1,127 @@
+parameters:
+  - name: azurecrName
+    type: string
+    default: pinscrsharedtoolinguks
+  - name: repositories
+    type: string
+  - name: slotName
+    type: string
+    default: staging
+
+steps:
+  - script: |
+      # Variables
+      REPOS=$(echo '${{ parameters.repositories }}' | jq -r '.[]')
+      echo "Deleting for: ${REPOS[@]}"
+
+      ACR_SUBSCRIPTION="edb1ff78-90da-4901-a497-7e79f966f8e2"
+      APP_SERVICE_SUBSCRIPTIONS=("962e477c-0f3b-4372-97fc-a198a58e259e" "76cf28c6-6fda-42f1-bcd9-6d7dbed704ef" "dbfbfbbf-eb6f-457b-9c0c-fe3a071975bc" "d1d6c393-2fe3-40af-ac27-f5b6bad36735")
```

Comment:
Azure DevOps Pipelines - Appeals Front Office - CI/CD
Needs access to view web apps in the Training environment, will fail currently